deepeval/synthesizer/context_generator.py

```diff
@@ -0,0 +1,186 @@
+from deepeval.synthesizer.doc_chunker import DocumentChunker, Chunk, get_embedding_similarity
```

Comment:
For imports, try to keep with current structure (first import everything outside of deepeval, then an empty line, before importing the rest from deepeval)